spec/unit/sneakers_spec.rb

```diff
@@ -0,0 +1,47 @@
+require 'spec_helper'
+require 'sneakers'
+require 'airbrake/sneakers'
+
+RSpec.describe Airbrake::Sneakers::ErrorReporter do
+  let(:worker) do
+    queue = instance_double('queue')
+    allow(queue).to receive_messages(
+      name: 'test-queue',
+      opts: {},
+      exchange: instance_double('exchange')
+    )
+
+    Sneakers.clear!
+    Sneakers.configure(daemonize: true, log: '/dev/null')
+    Sneakers::Worker.configure_metrics
+    Class.new do
+      include Sneakers::Worker
+      from_queue 'defaults'
+
+      def work(_)
+        raise 'oops error'
+      end
+    end.new(queue, Concurrent::ImmediateExecutor.new)
```

Comment:
It seems like second parameter is optional. Can we omit it?

Reply:
it seems we can